package.json

```diff
@@ -103,6 +103,8 @@
     "test:watch": "cross-env NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --no-single-run",
     "verify": "concurrently 'npm:test' 'npm:lint'",
     "jsdoc": "jsdoc -c jsdoc.json -R API.md -r -d dist/docs/api",
+    "update-about-dialog-copyright": "perl -pi -e 's/20\\d\\d\\-202\\d/2014\\-2022/gm' ./src/ui/layout/AboutDialog.vue",
```

Comment:
so if this uses perl does that mean the README should be updated to reflect this? most linux have it, but windows doesn't

Reply:
I think perl does come with WSL FWIW:
https://docs.microsoft.com/en-us/windows/wsl/install

but thank you for bringing this up! That said, this will get run max once a year by a developer, so hopefully won't be an issue for the README.

Reply:
I considered sed/grep, but they don't work the same on linux/osx. Were you thinking a node library?

Reply:
Oh oops. my bad. This is a script/